src/ioport.cpp

```diff
@@ -275,7 +275,7 @@ void UdpIOPort::importXml(ticpp::Element* pConfig)
         addr.sin_family = AF_INET;
         addr.sin_port = htons(rxport_m);
         addr.sin_addr.s_addr = htonl(INADDR_ANY);
-        if (bind(sockfd_m, (struct sockaddr *)&addr,sizeof(addr)) < 0) /* error */
+        if (bind(sockfd_m, reinterpret_cast<struct sockaddr *>(&addr),sizeof(addr)) < 0) /* error */
```

Comment:
AFAIK, casting is well defined in C++ and follows a specific order. You can't use the wrong one (reinterpret vs const for example) since it will error out.

I don't think there's even a size difference in compiled size since the compiler does this internally.